### PR TITLE
Management Dashboards

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR add monthly forms for having management indicators for : 
- Hospitalization
- Staff
- Finance

Indicators can be viewed through dashboards.

# Dashboards

## Hospitalization dashboard
image

Details on click on an indicator : 
image

## Finance Dashboard
image

## Staff Dashboard 
image

# Monthly Forms

## Registry
image

## Hospitalization Form
image

## Staff Form
image

## Finance Form
image

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles could we reassign this review to another developers ?

**Comment by jniles** [comment]
Yes sir - I am on vacation.  You shouldn't assume that I am available for reviews :).

File: client/src/i18n/fr/dashboard.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
+{
+  "DASHBOARD":{
+    "INDICATORS_FILES":{
+      "ALREADY_EXISTS":"La fiche des indicateurs pour cette periode existe déjà",
+      "FISCAL_YEAR":"Année fiscale",
+      "PERIOD":"Periode",
+      "SERVICE":"Service",
+      "TYPE":"Type de fiche",
+      "STATUS":"Etat",
```
Comment (lomamech) [inline client/src/i18n/fr/dashboard.json]: You must use 'État' instead of 'Etat'

File: client/src/i18n/fr/dashboard.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      "VALIDATED":"Validé",
+      "LAST_EDIT":"Modification",
+      "OVERVIEW":"Aperçu",
+      "PRINT":"Imprimer",
+      "DELETE":"Supprimer",
+      "EDIT":"Modifier",
+      "NB_DAYS":"Nombre des jours",
+      "NB_PERIODS":"Nombre des periodes",
```
Comment (lomamech) [inline client/src/i18n/fr/dashboard.json]: You must use 'Période' instead of 'Period'

File: server/models/bhima.sql [hunk 3]
```diff
@@ -117,8 +117,12 @@ INSERT INTO unit VALUES
   (229, 'Visits Registry', 'TREE.VISITS_REGISTRY', 'Visits registry', 12, '/modules/patient/visits', '/patients/visits'),
```
Comment (lomamech) [inline server/models/bhima.sql:117]: @mbayopanda  Please add all changes and additions of the database to the migrate file in order to facilitate the deployments of its various modules in the development sites

**Review by lomamech — COMMENTED** [review]
@mbayopanda Please add all changes and additions of the database to the migrate file in order to facilitate the deployments of its various modules in the development sites

**Review by lomamech — COMMENTED** [review]
@mbayopanda  
You have omitted the addition of script from the creation of the following tables
`indicator_type`, `hospitalization_indicator`, `staff_indicator`

but also the insertion of the data into the table
indicator_type

**Comment by lomamech** [comment]
@mbayopanda I think he should have a field to enter the number of surgery as well as the number of doctors (Number of Surgery or the number of doctors practicing Surgery)

image

**Comment by lomamech** [comment]
I think for the financial indicators, we should have the currency of the company
image

**Comment by mbayopanda** [comment]
Thank you @lomamech for your review 👍

**Comment by lomamech** [comment]
@mbayopanda  The standard should be in red, the green color is only for the percentage less than 8 percent
image

**Comment by lomamech** [comment]
just by the fact that we introduce the data per period, it would also be preferable to select only the periods instead of dates

**Comment by lomamech** [comment]
I also noticed that when looking at the data for a whole year the total number of days is 364 instead of 365 ??
image

**Comment by lomamech** [comment]
Regarding the bed occupancy rate, there is a problem regarding the number of beds, as the data are recorded by services, the total number of beds should be equivalent to the sum of bed number of each service for the last period recorded

**Pull request merged**